docs/accelerators.md

```diff
@@ -214,3 +214,16 @@ THROUGHPUT
 ```
 
 > **NOTE**: currently, AUTO plugin cannot be used with `--shape auto` parameter while GPU device is enabled.
+
+## Using NVIDIA Plugin
+
+*Note:* To build container with NVIDIA plugin use command: 
+```bash
+   OV_SOURCE_BRANCH=2022.1.0 NVIDIA=1 OV_USE_BINARY=0 make docker_build
```

Comment:
In other places we start with `make` and argument come next. Can we keep that convention here as well?